src/platform/packages/shared/response-ops/rule_params/synthetics_monitor_status/v1.ts

```diff
@@ -30,6 +30,11 @@ const NumberOfChecksSchema = schema.object({
   }),
 });
 
+const AlertOnNoDataSchema = schema.object({
+  // This parameters is not used in the current implementation but is kept for future use if the user wants to wait for a certain number of missing pings before alerting
+  noOfMissingPings: schema.number({ min: 1 }),
```

Comment:
Is this a parameter that users need to provide when creating a rule via API? If it is not used, I think it can be confusing when someone creates a rule using our APIs directly.

Reply:
I left that comment because the implementation details of this issue were not very clear and I was waiting for @shahzad31 to discuss if we want to set a minimum number of missing pings to fire the alert or if we want to trigger the alert as soon as 1 ping is missing. 

In the first case I need to change the implementation, in the second case this parameter can become just a boolean.

In both cases I'd keep the AlertOnNoDataSchema an object for future improvements.

Reply:
In the end this was changed to be just a boolean [here](https://github.com/elastic/kibana/pull/220127/commits/4b210f481dd0d9c719773e63d6aa164a86a3b0a8)